### ISSUE TITLE
[Bug]: Unfortunately the plot could not be edited. No plot object was found

### ISSUE THREAD

**Issue by ataylor92110** [jasp-stats/jasp-issues#2087]
### JASP Version

0.16.4

### JASP Module

ANOVA

### What analysis are you seeing the problem on?

2x2 Anova

### What OS are you seeing the problem on?

Windows 11

### Bug Description

[Combined New Data Set S23.zip](https://github.com/jasp-stats/jasp-issues/files/11185942/Combined.New.Data.Set.S23.zip)
When I tried to edit a graph to make the y-axis go from highest to lowest possible value, and do that across all related graphs in the different analyses, I got the error message. I did this frequently in the past and never got this error message. 

### Expected Behaviour

I should have been able to edit the image. In this case I wanted the y-axis to go from -5 to 5 in 1 step increments. 

### Steps to Reproduce

1. I click on edit image.
2. It opens in a new window.
3. I click on y-axis
4. I click on specify sequence
5. The bug shows up
![jasp error 2](https://user-images.githubusercontent.com/55157337/230786776-a4761b75-edcf-4857-afaf-659ce194384e.png)

6. I then exit and try again and I get this error: 
...
![jasp error 1](https://user-images.githubusercontent.com/55157337/230786821-873ecd66-8e88-4e02-b986-9ab028044fd1.png)

### Log (if any)

[JASP 2023-04-09 10_03_25 Engine 0.log](https://github.com/jasp-stats/jasp-issues/files/11185987/JASP.2023-04-09.10_03_25.Engine.0.log)
[JASP 2023-04-09 10_03_25 Desktop.log](https://github.com/jasp-stats/jasp-issues/files/11185988/JASP.2023-04-09.10_03_25.Desktop.log)

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by tomtomme** [comment]
Can you try with jasp 0.17.1?

**Comment by ataylor92110** [comment]
Sure I will have to update. I also think I might be able to delete the
graphs completely and then try to regraph. But I'll try the update first. I
noticed that graphs that are newer are not showing this problem to adjust
the y-axis.

And I'm sorry I'm a bit lame about these things. Honestly, I went to
graduate school in the 1980's. Everything else I have had to teach myself.
I took a fortran class back in the mid-80's :-) and that was my last forma
instruction in programming.

So I often get stymied these days ;-)

Thanks for the advice and I will let you know if this works.

Annette


Annette Kujawski Taylor, Ph.D.

she/her/hers ("What Are Personal Pronouns & Why Do They Matter?"
<https://www.mypronouns.org/what-and-why>)
Professor, Emeritus, Psychological Sciences
University of San Diego
***@***.***

**Comment by ataylor92110** [comment]
Ok. I installed 0.17 but it took over an hour for the file to load after
that! I thought it was not going to load at all but I decided to just leave
it and go for a walk ;-) My laptop shuts itself down after 15 minutes of
nonuse by the processor and it was still on when I returned and the file
was open so I assume it took about an hour to load but OK. It's a big file.

I lost several analyses in the transition -- primarily t-tests so easy
enough to redo; and I am going to attach new error messages.

But it did not solve the problem for all the graphs. Some of them yes, and
some of them no. I am still going to try deleting the analysis and then
redoing the analysis to see if that works.

And I am sending the logs as files because it's easier for now. I hope that
is OK.

Annette


Annette Kujawski Taylor, Ph.D.

she/her/hers ("What Are Personal Pronouns & Why Do They Matter?"
<https://www.mypronouns.org/what-and-why>)
Professor, Emeritus, Psychological Sciences
University of San Diego
***@***.***

**Comment by boutinb** [comment]
Hi @ataylor92110 , what could help up is to get a JASP file with just one analysis where this problem occurs.

**Comment by boutinb** [comment]
Hi @ataylor92110 It could be that you have a JASP file too big for your laptop to cope with. What I would like to know is if you just load the data file, and make one analysis, do you have still this problem with the plot editor? If yes, then just zip the jasp file and you can add it in this issue by using drag & drop.

**Comment by ataylor92110** [comment]
Thank you. I think you are probably correct. Now I cannot do
anything with this file.

But here is my question: why was it not too big for my laptop to handle
BEFORE I "upgraded" to version 0.17.xxxx? Except for being unable to edit
graphs there were no problems.

What I can do is potentially split it into two files: use the same data set
but do all of the anovas in one and all the others (descriptives,
frequencies, chi square and t tests) in another.

Sadly, I NEED the graphs to submit with the manuscript but I guess I can
export the data into excel and make excel graphs. What I like about JASP is
that it includes error bars but with excel I have to figure those out and I
find it to not be so straightforward.

What I am attaching for now are two screenshots of the error messages which
might support the idea that this is too large a file for my processor to
handle.

The reason I attached the task manager was simply that it might provide
more insight for you than for me on the overload on the system.

In the meantime I will try to figure out the other zip file. Sigh. I wish
these things were easier for me to figure out (e.g. even figuring out how
to zip files is complicated for me every time I have to do it. I have no
transfer of learning. Each time I have to relearn it from scratch; IDK why
but that's just how it is. Exhausting.)

**Comment by boutinb** [comment]
Hi @ataylor92110 I understand this is quite annoying. On my laptop I can load your jasp file, refresh the analyses and edit the plots without problem. Do you have maybe access to a computer with more power? What I could do also, is to chuck your JASP file in 3 or 4 pieces, so that you can use it on your computer.

**Comment by JorisGoosen** [comment]
@boutinb can you send me the link or jaspfile? Or @ataylor92110 of course!

boutinb went on vacation and I would like to help out on this issue. But I have no access to any of the logs or jaspfiles as they arent mentioned here in the thread

**Comment by ataylor92110** [comment]
Thanks. I will do that this weekend.

Annette

Annette Kujawski Taylor, Ph.D.

she/her/hers ("What Are Personal Pronouns & Why Do They Matter?"
<https://www.mypronouns.org/what-and-why>)
Professor, Emeritus, Psychological Sciences
University of San Diego
***@***.***

**Comment by tomtomme** [comment]
@ataylor92110 
I tested with jasp 0.18.3 and your file that you uploaded to the very first post here. It was big with like 50 analysis or so. And I could edit images no problem.
If not fixed on your side, feel free to reopen

**Comment by ataylor92110** [comment]
THANKS. It was some time ago and I did work it out. For some reason, my
whole laptop was freezing whenever I tried to modify a graph, but whatever
the problem, it is no longer. I think I might have needed to reboot!

The simplest of all solutions to try first...

Annette

Annette Kujawski Taylor, Ph.D.

she/her/hers ("What Are Personal Pronouns & Why Do They Matter?"
<https://www.mypronouns.org/what-and-why>)
Professor, Emerita, Psychological Sciences
University of San Diego
***@***.***
